bsp_sedi/drivers/pm/aon/ipapg.S

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (c) 2023 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
BSD license should be used. See other bsp drivers.

chrome license header should be kept.